core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -1956,6 +1956,12 @@ package object config {
       .intConf
       .createWithDefault(10)
 
+  private[spark] val RDD_INITIAL_NUM_PARTITIONS =
```

Comment:
Can we change this to `RDD_LIMIT_INITIAL_NUM_PARTITIONS` in order to be consistent with the `RDD_LIMIT_SCALE_UP_FACTOR` configuration defined below?